app/jobs/cache/browser_channels/responses_for_prefix.rb

```diff
@@ -23,6 +23,17 @@ def perform(prefix)
   def generate_brotli_encoded_channel_response(prefix:)
     @site_banner_lookups = SiteBannerLookup.where("sha2_base16 LIKE ?", prefix + "%")
     channel_responses = PublishersPb::ChannelResponseList.new
+
+    parameters = Rewards::Parameters.new.get_parameters
+
+    case parameters
+    when Rewards::Types::ParametersResponse
+      allowed_regions = parameters.custodianRegions
+    else
+      LogException.perform(failure)
```

Comment:
@jlbyrne If you just raise the error it will automatically get logged to newrelic.

Reply:
Other than that looks good.